client/src/styles/_alert.scss

```diff
@@ -15,27 +16,34 @@
 
   &::before {
     content: "";
-    width: 1.5rem;
+    width: var(--color-bar-width);
     position: absolute;
     top: 0;
     left: 0;
     bottom: 0;
-    border-radius: 0.4rem 0 0 0.4rem;
+    border-radius: 0.3rem 0 0 0.3rem;
   }
 
-  div.jf-alert__content {
+  .jf-alert__content {
     margin-right: auto;
+    padding-top: 1rem;
+    &:last-child {
+      padding-right: 1rem;
+    }
   }
 
-  button.jf-alert__close {
+  .jf-alert__close {
     align-self: flex-start;
+    // TODO: target area of button is still less than 44x44 px for a11y
```

Comment:
is there a ticket for this? if so i'd err on the side of not leaving in `TODO` comments

Reply:
yeah good point - added a shortcut card to the design system collection and will remove this and others.